commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/CommandAPIBukkit.java

```diff
@@ -409,6 +409,13 @@ public BukkitCommandSender<? extends CommandSender> wrapCommandSender(CommandSen
 				// We literally cannot type this at compile-time, so let's use a placeholder CommandSender instance
 				return new BukkitFeedbackForwardingCommandSender<CommandSender>(FeedbackForwardingSender.cast(sender));
 			}
+
+			final Class<? extends CommandSender> NullCommandSender = paper.getNullCommandSender();
+			if (NullCommandSender.isInstance(sender)) {
```

Comment:
One small concern: I spot an NPE here. If we're running a version of Paper that does not include the NullCommandSender class (anything before 1.20.6), then `isPaperPresent` will be true, but `nullCommandSender` will be null. So, `NullCommandSender#isInstance` will NPE.

This _would_ only happen if a CommandSender implementation that does not match any of the previous cases was passed into here. That probably wouldn't happen, but if it did, we want the exception to be the `Failed to wrap CommandSender` exception thrown below instead of a general NPE.

So maybe add a `null` check here (and a note to the javadoc)? It's a little weird when Paper-specific features are _also_ version dependent, but I think this is fine.

```suggestion
			final Class<? extends CommandSender> NullCommandSender = paper.getNullCommandSender();
			if (NullCommandSender != null && NullCommandSender.isInstance(sender)) {
```